src/ol/interaction/modifyinteraction.js

```diff
@@ -27,6 +28,60 @@ goog.require('ol.style.Style');
 
 
 /**
+ * @enum {string}
+ */
+ol.ModifyEventType = {
+  /**
+   * Triggered upon feature modification start
+   * @event ol.ModifyEvent#modifystart
+   * @api stable
```

Comment:
I wouldn't start this as stable? We normally add things first as experimental, and then make them stable later on.